src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1386,6 +1387,31 @@ internal override IEnumerable<FieldSymbol> GetFieldsToEmit()
                             yield return associatedField;
                         }
                         break;
+                    case SymbolKind.Property:
+                        if (m is SourcePropertySymbolBase propertySymbol)
+                        {
+                            // Ensure the binding is done so we guarantee that we have the BackingField set if necessary.
```

Comment:
>// Ensure the binding is done so we guarantee that we have the BackingField set if necessary. [](http://example.com/codeflow?start=28&length=93)

This logic is probably better to encapsulate in ```SourcePropertySymbolBase```.

Reply:
@AlekseyTs Won't that cause us to force-bind accessors potentially unnecessarily?